packages/core/src/utils/checkFormInputValue.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Checks if the value passed to FormInput is different than
+ * null, undefined or empty strings.
+ *
+ * Returns an boolean related to whether
+ * the field is empty or not
+ *
+ * @param value Value prop passed to FormInput
+ */
+const checkFormInputValue = (value: any) => { // eslint-disable-line
```

Comment:
If this returns a `bool` we should use ts to indicate it

```suggestion
const checkFormInputValue = (value: any): boolean => { // eslint-disable-line
```